.github/actions/kind-external-network/action.sh

```diff
@@ -42,6 +42,7 @@ lvh_wrapper docker network create \
     --driver bridge \
     -o com.docker.network.bridge.enable_ip_masquerade=true \
     -o com.docker.network.driver.mtu=$MTU \
+    -o com.docker.network.bridge.name=br-$KINDNETWORK \
```

Comment:
Why is this fix required, is this due to a previous change in the PR, or is it currently broken in `main`?

Reply:
>  maybe you have an idea why setting the bridge name could be important for checksums o_O

Wut. I have no clue. Does not sound like it should be related :cry: